tests/test_network_factory.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from parameterized import parameterized
+
+from monai.networks.nets import DenseNet, DenseNet121, NetworkFactory, UNet
+
+TESTS = [
+    [
+        UNet,
+        "UNet",
+        {"dimensions": 3, "in_channels": 4, "out_channels": 2, "channels": (4, 8, 16, 32), "strides": (2, 4, 1)},
+    ],
+    [DenseNet, "DenseNet", {"spatial_dims": 2, "in_channels": 4, "out_channels": 2}],
+]
+
+
+class TestNetworkFactory(unittest.TestCase):
+    @parameterized.expand(TESTS)
+    def test_network_factory(self, class_name, name, args):
+        net = NetworkFactory.get_constructor(name, **args)
```

Comment:
If anyone could help, I can't figure out how to do `NetworkFactory[name](**kwargs)` or `NetworkFactory[name, **kwargs]` or something similar to construct the object the network with kwargs.

Reply:
Hi @ericspod , could you please help here as you defined the other factories before?

Thanks.